infinity/test_infinity.py

```diff
@@ -8,6 +8,7 @@
 import random
 import string
 import contextlib
+from infinity._infinity import ClientConfig
```

Comment:
from python, do not import ClientConfig from .so, check __init__.py

Reply:
fixed